### PR TITLE
Update man page and README to clarify the --group option

### PR DESCRIPTION
**Pull request by m8ram**
This update is a first step to address #156 it simply updates the man page and the README to clarify what the option currently does.

### CODE REVIEW

**Review by ahitacat — CHANGES_REQUESTED** [review]
I will also suggest to make such changes in insights-core. In insights-core are defined the parameters of the cli tool, so when use the `--help` parameter it will show this information update and correct.

File: README.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ Command line switches available and their explanations.
 
 - `--register` - Register a system with the Insights API. Required for basic collection & upload, except in the case of certain targets or if `--offline` is specified
 - `--display-name=DISPLAYNAME` - Display name to appear in the insights web UI. This can be used at registration time, or standalone to change the display name any time.
-- `--group=GROUP` - Group to add the system to in the insights web UI. Only used on machine registration.
+- `--group=GROUP` - Add a tage named group to the system in the insights web UI.
```
Comment (ahitacat) [inline README.md:151]:
```suggestion
- `--group=GROUP` - Add a tag named `group` to the system in the insights web UI and in insights-client tags file.
```

File: docs/insights-client.8 [hunk 2]
```diff
@@ -19,7 +19,7 @@ Unregister host from the Red Hat Insights service.
 .IP "--display-name=DISPLAYNAME"
 Display name to use for this host. May be used during registration.
 .IP "--group=GROUP"
-Group to add this host to. The group will be saved to the tags file.
+Add a tag name group with the specified value to this host. The tag will be saved to the tags file.
```
Comment (ahitacat) [inline docs/insights-client.8:22]:
```suggestion
Add a tag named `group` with the specified value to this host. The tag will be saved to the tags file.
```

**Review by ahitacat — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ptoscano) [inline README.md:151]: also typo "tage"

**Review by ptoscano — APPROVED** [review]
LGTM, thanks!

**Pull request merged**